src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -257,7 +257,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Shared Function BeginsBaseLiteral(c As Char) As Boolean
             Return (c = "H"c Or c = "O"c Or c = "B"c Or c = "h"c Or c = "o"c Or c = "b"c) OrElse
-                    (IsFullWidth(c) AndAlso (c = FULLWIDTH_LATIN_CAPITAL_LETTER_H Or c = FULLWIDTH_LATIN_CAPITAL_LETTER_O Or c = FULLWIDTH_LATIN_CAPITAL_LETTER_B Or c = FULLWIDTH_LATIN_SMALL_LETTER_H Or c = FULLWIDTH_LATIN_SMALL_LETTER_O Or c = FULLWIDTH_LATIN_CAPITAL_LETTER_B))
```

Comment:
>FULLWIDTH_LATIN_CAPITAL_LETTER_B [](start = 245, length = 32)

I think we should simply use the right constant here avoid doing unnecessary refactoring.

Reply:
Refactored down to 2 functions the original for half width and another for the full width.

Reply:
@AlekseyTs updated

Reply:
@AlekseyTs I still prefer the two function approach, but I've inline the contents.